packages/react/src/components/DataTable/TableHead.tsx

```diff
@@ -5,9 +5,12 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import React, { ThHTMLAttributes } from 'react';
 import wrapComponent from '../../tools/wrapComponent';
 
-const TableHead = wrapComponent({
+export type TableHeadProps = ThHTMLAttributes<HTMLTableCellElement>;
```

Comment:
should this be an `HTMLTableSectionElement` instead? https://developer.mozilla.org/en-US/docs/Web/API/HTMLTableSectionElement. Doesn't look like `thead` is considered an HTMLTableCellElement https://developer.mozilla.org/en-US/docs/Web/API/HTMLTableCellElement
```suggestion
export type TableHeadProps = ThHTMLAttributes<HTMLTableSectionElement>;
```

Reply:
Thanks for pointing it out. I have updated the code.